### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.3.3 Sensory Characteristics

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#648]
(Mary Jo) 

- If the WCAG understanding gets updated or a WCAG2ICT note is added, this would resolve MATF issue #24 without them having to add anything specific for mobile.

(Chris L)

- If MATF is intending to add the techniques, that would go against purpose of the MATF document that states it doesn't provide techniques on how to meet.

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
